build.sbt

```diff
@@ -127,21 +136,51 @@ val postgresV = "42.2.5"
 val flywayV = "5.2.4"
 val confluentV = "4.1.2"
 
+lazy val dockerSettings = {
+  val workingDir = "/opt/nussknacker"
+
+  Seq(
+    dockerEntrypoint := Seq(s"$workingDir/bin/docker-run.sh", dockerPort.toString),
+    dockerExposedPorts := Seq(dockerPort),
+    dockerExposedVolumes := Seq(
+      "/opt/flinkData/savepoints",
+      s"$workingDir/logs",
+      s"$workingDir/db"
+    ),
+    defaultLinuxInstallLocation in Docker := workingDir,
+    dockerBaseImage := "openjdk:8-jdk",
+    dockerUsername := dockerUserName,
+    packageName := dockerPackageName,
+    dockerUpdateLatest := dockerUpLatest,
+    dockerLabels := Map(
```

Comment:
scala & flink versions also should be in tags?

Reply:
:) Version of application can be different than docker tag. I mean that docker-latest docker tag can store many of versions.

Reply:
I'v added a scala version, we can think about flink version.